commit 49ceb20d487f52de1c6ff3fafe95053e82a3ac23 (refs/changes/21/1521/1)
Author: Nick Zivkovic <nick.zivkovic@joyent.com>
Date:   2017-02-15T23:19:59+00:00 (2 years, 8 months ago)
    
    NAPI-394 update ProvisionNic docs for 'cn_uuid' to mention being required for fabric networks

diff --git a/docs/index.md b/docs/index.md
index 26cb66e..ef34931 100644
--- a/docs/index.md
+++ b/docs/index.md
@@ -448,20 +448,22 @@ No response payload, only a "204 No Content" response status.
 
 ## ProvisionNic (POST /networks/:network_uuid/nics)
 
-Creates a new nic, provisioning an IP and MAC address in the process.
+Creates a new nic, provisioning an IP and MAC address in the process. If
+creating a NIC on a fabric-network, a `cn_uuid` must be supplied as well.
 
 ### Inputs
 
 | Field             | Type                   | Description                                                                      |
 | ----------------- | ---------------------- | -------------------------------------------------------------------------------- |
 | owner_uuid        | UUID                   | Nic Owner                                                                        |
+| cn_uuid  |        | UUID                   | The UUID of the CN this NIC is on (optional, unless on frabric)                  |
 | belongs_to_uuid   | UUID                   | The UUID of what this Nic belongs to                                             |
 | belongs_to_type   | String                 | The type that this belongs to (eg: 'zone', 'server')                             |
 | ip                | String                 | IP address to assign to the nic                                                  |
 | reserved          | Boolean                | Whether the IP address should be reserved                                        |
 | nic_tags_provided | Array of nic tag names | Nic tags this nic provides                                                       |
 | check_owner       | Boolean                | If set to false, skips network ownership checks (optional)                       |
-| status            | String                 | Set state nic starts in (one of 'provisioning', 'stopped', 'running') (optional) |
+| state             | String                 | Set state nic starts in (one of 'provisioning', 'stopped', 'running') (optional) |
 
 **Notes:**
 
